Extensions/Electron/JsExtension.js

```diff
@@ -0,0 +1,603 @@
+        _('Enable content protection: _PARAM0_'),
+        _('Advanced window tools/Windows, Linux, MacOS'),
+        'res/actions/window24.png',
+        'res/actions/window.png'
+      )
+      .addParameter('yesorno', _('Enable content protection?'), '', false)
+      .setDefaultValue('true')
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Electron/electron-advancedwindowtools.js')
+      .setFunctionName('gdjs.evtTools.electron.setContentProtection');
+
+    extension
+      .addAction(
+        'SetFocusable',
+        _('Allow focusing'),
+        _('Allow or disallow the user to focus the window.'),
+        _('Allow window focus: _PARAM0_'),
+        _('Advanced window tools/Windows, Linux, MacOS'),
+        'res/actions/window24.png',
+        'res/actions/window.png'
+      )
+      .addParameter('yesorno', _('Allow focus?'), '', false)
+      .setDefaultValue('true')
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Electron/electron-advancedwindowtools.js')
+      .setFunctionName('gdjs.evtTools.electron.setFocusable');
+
+    extension
+      .addAction(
+        'Flash',
+        _('Flash the window'),
+        _('Make the window flash or end flashing.'),
+        _('Make the window flash: _PARAM0_'),
+        _('Advanced window tools/Windows, Linux, MacOS'),
+        'res/actions/window24.png',
+        'res/actions/window.png'
+      )
+      .addParameter('yesorno', _('Flash the window?'), '', false)
+      .setDefaultValue('true')
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Electron/electron-advancedwindowtools.js')
+      .setFunctionName('gdjs.evtTools.electron.flash');
+
+    extension
+      .addAction(
+        'SetOpacity',
+        _('Set window opacity'),
+        _('Changes the window opacity. The new opacity should be between 0 and 1.'),
+        _('Set the window opacity to _PARAM0_'),
+        _('Advanced window tools/Windows, Linux, MacOS'),
+        'res/actions/window24.png',
+        'res/actions/window.png'
+      )
+      .addParameter('number', _('New opacity'), '', false)
+      .setDefaultValue('true')
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Electron/electron-advancedwindowtools.js')
+      .setFunctionName('gdjs.evtTools.electron.setOpacity');
+
+    extension
+      .addAction(
+        'Center',
+        _('Center window'),
+        _('Centers the current window.'),
+        _('Center window'),
+        _('Advanced window tools/Windows, Linux, MacOS'),
+        'res/actions/window24.png',
+        'res/actions/window.png'
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Electron/electron-advancedwindowtools.js')
+      .setFunctionName('gdjs.evtTools.electron.center');
+
+    extension
+      .addAction(
+        'SetWindowPosition',
+        _('Set window position'),
+        _('Changes the window position.'),
+        _('Set the window position to _PARAM0_X and _PARAM0_Y'),
```

Comment:
Careful about the arguments, one is wrong.
Also, specify positions as `x;y`, like in the actions to set object position:

`Set the window position to _PARAM0_;_PARAM1_`